### ISSUE TITLE
trailing char help

### ISSUE THREAD

**Issue by tronmovie110011** [tinyMediaManager/tinyMediaManager#286]
Version: 2.9.10
Build: 2018-04-26 16:18
OS: Windows 7 6.1
JDK: 1.8.0_171 amd64 Oracle Corporation

__What is the actual behaviour?__
this happens when im loading Running man into the datasources

[tmm_logs.zip](https://github.com/tinyMediaManager/tinyMediaManager/files/1973492/tmm_logs.zip)

__What is the expected behaviour?__

__Steps to reproduce:__

__Additional__
Have you attached the logfile from the day it happened?

**Comment by mlaggner** [comment]
looks like your folder name has a trailing space in it and the windows file system parser inside java does not like that..
`C:\bd temp 2\The Running Man `

**Comment by tronmovie110011** [comment]
Yes that zip file is it. I did get rid of the space and it was okay but it seams that the main file folder is getting corrupt and not allowing me to change the name so I have to create a new folder and cut and paste the movie file in that then scrape the file. This just started to happen after I did the latest update of TMM

**Comment by mlaggner** [comment]
did you rename this folder via tmm?
we do have a deletion of trailing spaces in the renamer - this should not be caused by tmm

**Comment by myron0815** [comment]
Windows does not allow trailing spaces, and you cannot create them (easily).
Explorer always trims them, so Java simply does not support this.
see https://bugs.openjdk.java.net/browse/JDK-8190546

That you CAN have folders with trailing spaces (from an unix system or whatever) is a different story - but we cannot do anything about it :|
